specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-07-12-preview/AlertsManagement.json

```diff
@@ -0,0 +1,1733 @@
+        },
+        "monitorServices": {
+          "items": {
+            "$ref": "#/definitions/smartGroupAggregatedProperty"
+          },
+          "type": "array",
+          "description": "Summary of monitorServices in the smart group"
+        },
+        "monitorConditions": {
+          "items": {
+            "$ref": "#/definitions/smartGroupAggregatedProperty"
+          },
+          "type": "array",
+          "description": "Summary of monitorConditions in the smart group"
+        },
+        "alertStates": {
+          "items": {
+            "$ref": "#/definitions/smartGroupAggregatedProperty"
+          },
+          "type": "array",
+          "description": "Summary of alertStates in the smart group"
+        },
+        "alertSeverities": {
+          "items": {
+            "$ref": "#/definitions/smartGroupAggregatedProperty"
+          },
+          "type": "array",
+          "description": "Summary of alertSeverities in the smart group"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of alerts. Call ListNext() with this URI to fetch the next page alerts."
+        }
+      }
+    },
+    "smartGroupAggregatedProperty": {
+      "type": "object",
+      "description": "Aggregated property of each type",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the type."
+        },
+        "count": {
+          "type": "integer",
+          "description": "Total number of items of type."
+        }
+      }
+    },
+    "MonitorServiceList": {
+      "title": "Monitor service details",
+      "description": "Monitor service details",
+      "required": [
+        "data"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/alertsMetaDataProperties"
+        },
+        {
+          "type": "object"
+        }
+      ],
+      "properties": {
+        "data": {
+          "description": "Array of operations",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MonitorServiceDetails"
+          }
+        }
+      }
+    },
+    "MonitorServiceDetails": {
+      "description": "Details of a monitor service",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Monitor service name"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Monitor service display name"
+        }
+      }
+    },
+    "alertEnrichmentResponse": {
+      "description": "The alert's enrichments.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/alertEnrichmentProperties"
+        }
+      }
+    },
+    "alertEnrichmentsList": {
+      "description": "List the alert's enrichments.",
+      "properties": {
```

Comment:
>properties [](http://example.com/codeflow?start=7&length=10)

lack of  `nextLink`:

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/get-collection-only-has-value-nextlink.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/get-collection-only-has-value-nextlink.md)

Reply:
added.